multicast/hear.py

```
@@ -292,10 +532,14 @@ def server_bind(self):
 		Returns:
 			None
 		"""
-		print(str("server_bind"))
+		self.logger.info("server_bind")
 		super(McastServer, self).server_bind()
```

Ruff (0.8.2)

Use `super()` instead of `super(__class__, self)`

Remove `__super__` parameters

(UP008)